eslint.config.mjs

```diff
@@ -20,6 +20,7 @@ export default tseslint.config(
 			'**/dist/**',
 			'**/node_modules/**',
 			'**/__snapshots__/**',
+			'**/*.js',
```

Comment:
Do we still need this?

Reply:
yes since the output file is dist/index.js